PyRDF/backend/Backend.py

```diff
@@ -1,4 +1,10 @@
 class Backend(object):
+    """
+    A Generic Backend class (Interface)
```

Comment:
"""
Base class for RDataFrame backends.
Subclasses need to implement 'execute'.
"""